src/app/models/document.ts

```diff
@@ -48,6 +49,7 @@ export class Document {
     this.dateUploaded = obj && obj.dateUploaded || null;
     this.dateUpdated = obj && obj.dateUpdated || null;
     this.datePosted = obj && obj.datePosted || null;
+    this.dateAdded = obj && obj.dateAdded || null;
```

Comment:
I know this is probably related to previous work but what is this `dateAdded` and how is it different from `dateUploaded` and `datePosted`?